.github/workflows/type-checking.yaml

```diff
@@ -5,31 +5,33 @@ on:
     branches:
       - main
   pull_request:
-    paths-ignore:
-      - "doc/**"
-      - README.md
+    paths:
+      - yt/**/*.py
```

Comment:
Do we only want py, or should we also include pyx, pxi, etc?  Does mypy do anything there?